src/blocks/author-profile/components/inspector/ContentPanel.js

```
@@ -0,0 +1,183 @@
+								label={__('Facebook', 'author-profile-blocks')}
+								checked={socialLinksToShow?.includes(
+									'facebook'
+								)}
+								onChange={(checked) => {
+									const currentLinks =
+										socialLinksToShow || [];
+									setAttributes({
+										socialLinksToShow: checked
+											? [...currentLinks, 'facebook']
+											: currentLinks.filter(
+													(link) =>
```

ESLint

[error] Expected indentation of 12 tabs but found 13.

(indent)